src/super_gradients/training/losses/cwd_loss.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional
+
+from torch.nn.modules.loss import _Loss
+import torch.nn as nn
+import torch
+
+
+class CWDKlDivLoss(_Loss):
```

Comment:
CWDKlDivLoss is a bit hard to read and I wonder why not ChannelWiseKnowledgeDistillationLoss ?

Reply:
Renamed [CR fixes](https://github.com/Deci-AI/super-gradients/pull/731/commits/59abdf6d228f14d71a74411ad85dac745bdd2efb)